### DISCUSSION TITLE
Welcome to PaperCutExamples Discussions!

### DISCUSSION THREAD

**Discussion by alecthegeek** [PaperCutSoftware/PaperCutExamples#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by toaddy-pc** [comment]
Hi Alec,
have really enjoyed your python scripts for papercut - starting to put them to use. With regards to the guide you have on configuring device settings such as card, username password etc I am wondering if there is device settings for the tracking functions copying ,faxing to be able to toggle them on/off?
Regards
John
